CONTRIBUTING.md

```diff
@@ -198,6 +198,26 @@ HEADLESS=false SLOW_MO=500 npm run wtest
 BROWSER=chromium node --inspect-brk test/test.js
 ```
 
+- When should a test be marked with `skip` or `fail`?
+
+  - **`skip(condition)`**: This test *should ***never*** work* for `condition`
+    where `condition` is usually a certain browser like `FFOX` (for Firefox),
+    `WEBKIT` (for WebKit), and `CHROMIUM` (for Chromium).
+
+    For example, the [alt-click downloads test](https://github.com/microsoft/playwright/blob/471ccc72d3f0847caa36f629b394a028c7750d93/test/download.spec.js#L86) is marked
```

Comment:
I think pointing to a concrete (permalinked) example is helpful, but this may throw people off about alt-click download once Playwright is fixed to support alt-click in WebKit and Chromium.

Consider adding a completely fiction example like:

```
it.fail(CHROMIUM)('should do exciting things', async({page, server} => {
     const excitingThings = await page.excitingThings()
     expect(excitingThings).toBeTruthy()
}))
```

Reply:
Yeah, that's a good point. On the other hand as long as it points to a particular revision of the file and not break over time I think it's ok. We can iterate on this later if it's unsatisfactory.